### ISSUE TITLE
Explanation For Hotspot Options

### ISSUE THREAD

**Issue by Desoroxxx** [stokito/IdeaJol#41]
![image](https://user-images.githubusercontent.com/82710983/234120515-254333dd-a578-4456-8109-236fd9e4d2af.png)

There are a lot of options, and I couldn't find an answer on what some of it means on the internet.

What is `>= 8` or `>= 15` I would guess that this would be the Java version but that is not clear since there is no J or Java (Could be entirely wrong).

What is `COOPS` and `CCPS` and `16-byte align`?

And for Lilliput I could find an answer online and it is remarkably interesting and makes me excited for the future of Java.

Hope you can help me, it would be interesting to add a link to your answer in the plugin somewhere even.

**Comment by stokito** [comment]
Yeah, I thinked to add and explanation and also to make a video. In short just leave default setting which is COPS with 8 byte align. Others are just for researchers.

In short COPS is when 32 bit pointers (4 bytes) are used on 64 bit CPUs.

The Liliput is an OpenJDK project to get rid off object headers but it's in early stage.

**Comment by stokito** [comment]
For all questions see Alexey Sipilev talk  In the readme here it must be a link as I remember

**Comment by stokito** [comment]
I added a tooltip and a separate reame section. Also added a link to documentation and videos
